### PR TITLE
Preprod: movie: Warning (2021) initial review

### PR DESCRIPTION
**Pull request by paul-sheridan**
A disconnected mishmash of sci-fi tropes and philosophical musings. Rating 1/5.

### CODE REVIEW

File: notes/stories/film/film-scifi-2020s.st.txt [hunk 1]
```diff
@@ -1308,6 +1308,53 @@ revival long after death [It was revealed that Neo and Trinity had been resurrec
...
+https://en.wikipedia.org/wiki/Warning_(2021_film)
+
+:: Ratings
+1 <paul>
+
+:: Major Themes
+asteroid impact [The film culminates with the Earth being destroyed by incoming asteroids.]
+god and follower [One storyline follows Claire and her complete and utter dependence on a "God" model virtual assistant that gives her moral guidance and records all her sins.]
+coping with being lost in space [One storyline follows an android that ends up floating aimlessly in space. The android mused about existence and ultimately struck up a one-way conversation with God upon seeing the Earth destroyed by asteroid collisions.]
+android [One storyline follows an android that was left floating aimlessly in space after an accident repairing a satellite. In a minor storyline, and old and unwanted android was enthanized by being crushed under a metal slab.]
```
[the remark this answers is not in the document]
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2020s.st.txt:1338]: "euthanised" is British English.

File: notes/stories/film/film-scifi-2020s.st.txt [hunk 2]
```diff
@@ -1308,6 +1308,53 @@ revival long after death [It was revealed that Neo and Trinity had been resurrec
...
+:: Title
+Warning
+
+:: Description
+Warning is a 2021 science fiction thriller film directed by Agata Alexander
+in her directorial debut, from a screenplay by Alexander, Jason Kaye and Ro
+Michaelson.
+
+Synopsis: Various otherwise unconnected people go about living their lives
+up until a swarm of death-dealing asteroids crash into the Earth.
```
[the remark this answers is not in the document]
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2020s.st.txt:1323]: Check.

**Comment by odinlake** [comment]
Be that as it may: is “enthanized” Canadian English???

**Comment by paul-sheridan** [comment]
Oops. I see the typo now in “enthanized” - not Canuck English, lol. Fortunately, I inadvertently fixed it in the previous "minor update".

**Pull request merged**